docs/source/release_notes.rst

```diff
@@ -35,6 +35,7 @@ v1.4.0 Jan 10, 2022
         * Skip code coverage for specific dask usage lines (:pr:`1829`)
         * Increase minimum required numpy version to 1.21.0, scipy to 1.3.3, koalas to 1.8.1 (:pr:`1833`)
         * Remove pyyaml as a requirement (:pr:`1833`)
+        * Adds `__setitem__` method to overload `add_dataframe` method on EntitySet (:pr:`1862`)
```

Comment:
This needs to move to the Future Release section above. Also, in order to get code formatting to work correctly, you will need to use double back-tics around the methods.